README.md

```diff
@@ -77,27 +76,24 @@ npm run create-md -- <filename> "<relative-path>"
 npm run create-md -- textarea "native/inputs"
 ```
 
-This command will create:
+```bash
+npm run create-md -- "links-buttons" "how-to-test/components"
+```
+
+These commands will create:
 
 ```
 public/content/native/inputs/textarea.md
 ```
+```
+public/content/how-to-test/components/links-buttons.md
+```
 
 - If the folder does not exist, it will be created.
```

Comment:
Can you say instead:
"If the category folder, such as "inputs" does not exist, it will be created.